### ISSUE TITLE
Power button and thermal control of Helm on Armbian

### ISSUE THREAD

**Issue by kkanatcit** [HelmSecure/armbian-images#18]
Questions:

After flashing the firmware to Armbian, does the power button actually function? And if not, what is needed to make it do so? The LED on front does not activate, but I can see a green glow through the casing and rear grill holes of an LED on the board that does not shut off after pressing the power button numerous times/ways.

Secondly, I remember that there was an issue with Helm Servers overheating before a fix was issued. After flashing to Armbian I have found my unit to become unresponsive after a period of time and when I went to examine it, the unit was really really hot. Could you tell me what sort of thermal monitoring/throttling was implemented so that I can do so similarly now?

And I guess, it’d be nice if I could find out how to control the front LED as well.

**Comment by l3nticular** [comment]
More specifically, if we could get a schematic to the board we would be able to answer the led and power button questions on our own.

**Comment by l3nticular** [comment]
Mine was also overheating when I first got it with the official software. I have not seen this with Armbian but I don’t have any services running on it yet; it’s just been idling.

![D358A672-2BBC-40A1-BC3B-4212A481CB12](https://user-images.githubusercontent.com/11479790/211389014-97f6e2ae-7a33-480f-b52f-94d6bfb6c0dd.jpeg)

**Comment by l3nticular** [comment]
I know I had a battery cell issue on that unit where the actual cell (standard 18650 I think) was bad.  You could try removing the batteries and see if it solves your overheating.  Maybe if you could also get a log/graph of temperature vs load and see if it’s correlated or not?

**Comment by sir-pinecone** [comment]
I had a battery get so hot that it melted a region of the case! I second removing the batteries and seeing if that fixes the temp issue.

**Comment by l3nticular** [comment]
Yeah that happened to mine as well. The cell was bad when I removed it and checked the voltage.

**Comment by kkanatcit** [comment]
My temp issue happened back when I was trying to format my SSD. Since that got taken care of it’s been ok.

At this point I’ve got a functioning email server... but it’d still be nice to know how to control the front LED. Is there a simple/quick/easy way to find out how Armbian recognizes it?
